docs/scripting/trust_scripts/sys.breach.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: sys.breach
+---
+
+> breaches sys
```

Comment:
```suggestion
sys.breach breaches the caller's system.
```

Reply:
In this set of pr's (and possibly the last two), the descriptions are invariably in quotes or not. I'm not sure which style you'd prefer but we should settle on one or the other. (if it is quotes, please add the `> ` back)